pkg/inspector/image-inspector_test.go

```diff
@@ -0,0 +1,35 @@
+package inspector
+
+import (
+	"testing"
+)
+
+func TestGetAuthConfigs(t *testing.T) {
+
+	tests := map[string]struct {
+		dockercfg      string
+		username       string
+		passwordFile   string
+		len            int
+		shouldValidate bool
```

Comment:
Thanks, you are right. I will change it to shouldFail and add a test for failure.